### ISSUE TITLE
file-type <16.5.4 vulnerable to Infinite Loop via malformed MKV file

### ISSUE THREAD

**Issue by azu** [azu/validate-image-type#4]
file-type <16.5.4 has a vulerablity.

```
❯ npm audit --omit  dev
# npm audit report

file-type  <16.5.4
Severity: moderate
file-type vulnerable to Infinite Loop via malformed MKV file - https://github.com/advisories/GHSA-mhxj-85r3-2x55
fix available via `npm audit fix --force`
Will install file-type@17.1.3, which is a breaking change
node_modules/file-type
```

This vulerablity was fixed in following versions.

- [Release v17.1.3 · sindresorhus/file-type](https://github.com/sindresorhus/file-type/releases/tag/v17.1.3)
- [Release v16.5.4 · sindresorhus/file-type](https://github.com/sindresorhus/file-type/releases/tag/v16.5.4)

**Comment by azu** [comment]
file-type has a breaking change. We need to do breakchanges.

**Comment by azu** [comment]
fixed in #5
